### ISSUE TITLE
gemm in cntk which kernel is [1024,2,2,256] -> onnx.

### ISSUE THREAD

**Issue by VictorYXL** [Microsoft/MMdnn#481]
Platform (like ubuntu 16.04/win10):win10
Python version:3.6
Source framework with version (like Tensorflow 1.4.1 with GPU):cntk CPU
Destination framework with version (like CNTK 2.3 with GPU): onnx
Pre-trained model path (webpath or webdisk path):
DepthwiseSeparableConvLayer {inChannels, outChannels, strideDepthwise, bnTimeConst} = Sequential (
    GroupConvolutionalLayer {inChannels, inChannels, inChannels, (3:3), init="heNormal", stride = strideDepthwise, pad = true, bias=false} :
    BatchNormalizationLayer {spatialRank = 2, normalizationTimeConstant = bnTimeConst, useCntkEngine = false, initialScale = 1} :
    ReLU :
    ConvolutionalLayer {outChannels, (1:1), init = "heNormal", stride = 1, pad = false, bias = false} :
    BatchNormalizationLayer {spatialRank = 2, normalizationTimeConstant = bnTimeConst, useCntkEngine = false, initialScale = 1} :
    ReLU
)

## MobileNet
#  Reference: "MobileNets: Efficient Convolutional Neural Networks for Mobile Vision Applications" https://arxiv.org/pdf/1704.04861.pdf
MobileNetFace (features, labels, labelDim, bnTimeConst) = {
    fc1 = Sequential (
        ConvBNReLULayer {32, (3:3), 2, bnTimeConst} :
        DepthwiseSeparableConvLayer {32, 64, 1, bnTimeConst} :
        DepthwiseSeparableConvLayer {64, 128, 2, bnTimeConst} :
        DepthwiseSeparableConvLayer {128, 128, 1, bnTimeConst} :
        DepthwiseSeparableConvLayer {128, 256, 2, bnTimeConst} :
        DepthwiseSeparableConvLayer {256, 256, 1, bnTimeConst} :
        DepthwiseSeparableConvLayer {256, 512, 2, bnTimeConst} :

        # 5 stacks
        DepthwiseSeparableConvLayer {512, 512, 1, bnTimeConst} :
        DepthwiseSeparableConvLayer {512, 512, 1, bnTimeConst} :
        DepthwiseSeparableConvLayer {512, 512, 1, bnTimeConst} :
        DepthwiseSeparableConvLayer {512, 512, 1, bnTimeConst} :
        DepthwiseSeparableConvLayer {512, 512, 1, bnTimeConst} :

        DepthwiseSeparableConvLayer {512, 1024, 2, bnTimeConst} :
        DepthwiseSeparableConvLayer {1024, 1024, 1, bnTimeConst} :
        AveragePoolingLayer {(7: 7), stride = 1} :
        LinearLayer{256, init = 'gaussian', bias = false}
    )(features)
    normalizedFeature = FeatureNormalizeLayer{ featureNormalizeType = 2 }(fc1)
    z0 = AdditiveFullConnectionLayer{ 256, labelDim, init = 'glorotUniform', weightNormalize = true, bias = -0.35, annealBias = false }(labels, normalizedFeature)
    z = 60 * z0
}
Running scripts:
mmconvert -sf cntk -iw xxx.cntk -df onnx -om xxx.onnx.

The last gemm layer in cntk.
The input is [1,1024,2,2] and kernel is [1024,2,2,256].
But can not get the correct kernel in onnx.
